components/admin_console/admin_user_card/admin_user_card.jsx

```diff
@@ -22,8 +22,7 @@ const AdminUserCard = (props) => (
         <div className='AdminUserCard__header'>
             <ProfilePicture
                 src={Client4.getProfilePictureUrl(props.user.id, props.user.last_picture_update)}
-                width='134'
-                height='134'
+                size='xxl'
```

Comment:
@michaelgamble this is converted to 128 instead of 134. we can have another xxxl or something like that, but I would love to homogenize the size of the avatars. Currently I have:

- xs: 16px
- sm: 24px
- md: 32px (default)
- lg: 36px
- xl: 50px
- xxl: 128px

Reply:
Up to @michaelgamble on the size. 128px looks fine to me here, but I do see an alignment issue: 
<img width="377" alt="Screen Shot 2019-08-16 at 4 16 10 PM" src="https://user-images.githubusercontent.com/44858354/63202974-5bf99000-c041-11e9-89c7-7764c6c0564a.png">

Reply:
Up to @michaelgamble on the size. 128px looks fine to me here, but I do see an alignment issue: 
<img width="377" alt="Screen Shot 2019-08-16 at 4 16 10 PM" src="https://user-images.githubusercontent.com/44858354/63202974-5bf99000-c041-11e9-89c7-7764c6c0564a.png">